include/viewport.h

```diff
@@ -0,0 +1,44 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   viewport.h                                         :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: reasuke <reasuke@student.42tokyo.jp>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2025/05/08 19:58:04 by reasuke           #+#    #+#             */
+/*   Updated: 2025/05/10 01:38:59 by reasuke          ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef VIEWPORT_H
+# define VIEWPORT_H
+
+# include "vector.h"
+
+/**
+ * Viewport Configuration for Ray Tracing
+ *
+ * This viewport is configured in a right-handed coordinate system where:
+ * - The camera is positioned on the positive Z-axis looking toward -Z direction
+ * - The viewport plane is parallel to the XY plane
+ * - X-axis corresponds to the horizontal direction (right is positive)
+ * - Y-axis corresponds to the vertical direction (down is positive)
+ *
```

Comment:
suggestion to add a figure like this:
```suggestion
 *      y
 *      ^
 *      |     (right-handed)
 *      |
 *      |
 *      |/
 *    --o---------> x
 *     /|
 *    /
 *   /
 *  z
 *
```